Documentation/commands.md

```diff
@@ -54,6 +54,12 @@ The metadata service helps running apps introspect their execution environment a
 
 * [metadata-service](subcommands/metadata-service.md)
 
+## API Service
+
+The api service helps users to list/introspect their pods and images on the machine.
```

Comment:
The API service allows clients to list and inspect pods and images running under rkt.